.github/workflows/release.yml

```diff
@@ -43,6 +43,18 @@ jobs:
           git commit -m "Release extension version ${{ inputs.version }}"
 
           git push --set-upstream origin release/${{ inputs.version }}
+      
+      - name: Update language server
+        if: ${{ inputs.update-language-server == 'true' }}
```

Comment:
You know, I think I'm actually wrong about this, it definitely *used* to work that way, but checking [this issue](https://github.com/actions/runner/issues/1483) it looks like it might be fixed now? I think I was betrayed by outdated knowledge.

Reply:
I updated it and if it behaves unexpectedly we can always change it back